webview/src/experiments/components/table/Row.tsx

```diff
@@ -98,6 +127,18 @@ const getSingleSelectMenuOptions = (
       'Modify and Queue',
       MessageFromWebviewType.VARY_EXPERIMENT_PARAMS_AND_QUEUE,
       !isNotCheckpoint
+    ),
+    experimentMenuOption(
+      [id],
+      starred ? 'Unstar Experiment' : 'Star Experiment',
+      MessageFromWebviewType.TOGGLE_EXPERIMENT_STAR,
+      isWorkspace
+    ),
+    withId(
```

Comment:
[Q] Should there be dividers between the VARY_AND_ commands and toggle star and also remove?